pkg/client/resmgmt/lscc.go

```diff
@@ -39,6 +41,7 @@ type chaincodeDeployRequest struct {
 	Name       string
 	Path       string
 	Version    string
+	Lang       string
```

Comment:
type should be pb.ChaincodeSpec_Type

Reply:
```golang
ccds := &pb.ChaincodeDeploymentSpec{ChaincodeSpec: &pb.ChaincodeSpec{
		Type: pb.ChaincodeSpec_GOLANG, ChaincodeId: &pb.ChaincodeID{Name: chaincode.Name, Path: chaincode.Path, Version: chaincode.Version},
		Type: pb.ChaincodeSpec_Type(pb.ChaincodeSpec_Type_value[strings.ToUpper(chaincode.Lang)]), ChaincodeId: &pb.ChaincodeID{Name: chaincode.Name, Path: chaincode.Path, Version: chaincode.Version},
		Input: &pb.ChaincodeInput{Args: chaincode.Args}}}
```
Hi, there is type conversion here.

Reply:
Generally, API should be type safe. Client has to import the protobuf package already, for "CollConfig []*pb.CollectionConfig", so I see no reason for not having type safe interface.

Reply:
Thanks for answering.